python_modules/dagster/dagster/_core/remote_representation/external.py

```diff
@@ -518,6 +518,8 @@ def description(self):
 
     @property
     def owners(self) -> Optional[Sequence[str]]:
+        if self._job_ref_snap is not None:
+            return self._job_ref_snap.owners
```

Comment:
this will cause owners to become None incorrectly for users who don't upgrade to the latest version in their user code, and had owners before

Reply:
oh! Did not realize. That's fine then.